planet/models.py

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 """Manage data for requests and responses."""
 import copy
-import datetime
```

Comment:
From my comment bellow, this should go back in either as is or as `from datetime import datetime` - chef's choice!